presto-main/src/test/java/com/facebook/presto/operator/aggregation/TestMergeQuantileDigestFunction.java

```diff
@@ -22,20 +21,18 @@
 import com.facebook.presto.spi.type.TypeParameter;
 import com.google.common.collect.ImmutableList;
 import io.airlift.stats.QuantileDigest;
-import org.testng.annotations.Test;
 
 import java.util.List;
 import java.util.function.BiFunction;
 
-import static com.facebook.presto.operator.aggregation.AggregationTestUtils.assertAggregation;
 import static com.facebook.presto.spi.type.QuantileDigestParametricType.QDIGEST;
 import static io.airlift.slice.Slices.wrappedBuffer;
 import static java.util.Objects.requireNonNull;
 
 public class TestMergeQuantileDigestFunction
-        extends AbstractTestAggregationFunction
+        extends TestMergeStatisticalDigestFunction
 {
-    public static final BiFunction<Object, Object, Boolean> QDIGEST_EQUALITY = (actualBinary, expectedBinary) -> {
+    static final BiFunction<Object, Object, Boolean> QDIGEST_EQUALITY = (actualBinary, expectedBinary) -> {
```

Comment:
Why removing `public`?

Reply:
Because it can be kept package-private without affecting functionality. Switching it back to `public` though.